sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/util/SearchPagedResponse.java

```diff
@@ -81,7 +81,7 @@ private static String createContinuationToken(SimpleResponse<SearchDocumentsResu
      * @return The percentage of the index covered in the search request if {@code minimumCoverage} was set in the
      * request, otherwise {@code null}.
      */
-    public Double getCoverage() {
+    Double getCoverage() {
```

Comment:
Just to make clear, we are now hiding the per-page metadata. I don't have an issue with it, just wanted to make certain I understand this change.

Reply:
Talked offline. We won't provide page-level API